installing/installing-preparing.adoc

```diff
@@ -0,0 +1,86 @@
+== Selecting your cluster installation type
+Before you install a {product-title} cluster, you need to select the best installation instructions to follow. Think about your answers to the following questions to select the best option.
+
+[id="installing-preparing-install-manage"]
+=== Do you want to install and manage an {product-title} cluster yourself?
+
+If you want to install and manage {product-title} yourself, you can install it on the following platforms:
+
+* Amazon Web Services (AWS)
+* Microsoft Azure
+* Google Cloud Platform (GCP)
+* {rh-openstack}
+* {rh-virtualization}
+* IBM Z and LinuxONE
+* IBM Power
+* VMware vSphere
+* Bare metal or other platform agnostic infrastructure
+
+You can deploy an {product-title} 4 cluster to both on-premise data centers and to cloud hosting services, but all of the machines in your cluster must be in the same datacenter or cloud hosting service.
+
+If you want to use {product-title} but do not want to manage your cluster yourself, you have several managed service options. If you want a cluster that is fully managed by Red Hat, you can use link:https://www.openshift.com/products/dedicated/[OpenShift Dedicated] or link:https://www.openshift.com/products/online/[OpenShift Online]. You can also use OpenShift as a managed service on Azure, AWS, IBM Cloud, or Google Cloud. For more information about managed services, see the link:https://www.openshift.com/products[OpenShift Products] page.
+
+[id="installing-preparing-migrate"]
+=== Have you used {product-title} 3 and want to use {product-title} 4?
+
+If you used {product-title} 3 and want to try {product-title} 4, you need to understand how different {product-title} 4 is. {product-title} 4 weaves the Operators that comprise the program and the operating system that the platform runs on, {op-system-first}, together seamlessly. Instead of deploying machines and configuring their operating systems so that you can install {product-title} on them, the {op-system} operating system is an integral part of the {product-title} cluster. Your cluster deploys the operating system for the cluster machines as part of the program itself. The easiest way to use {product-title} is by letting it make all of the infrastructure and machines that it needs.
+
+Because you need to provision machines as part of the {product-title} cluster installation process, you cannot upgrade an {product-title} 3 cluster to {product-title} 4. Instead, you must create a new {product-title} 4 cluster and migrate your {product-title} 3 workloads to them. For more information about migrating, see link:https://redhat-cop.github.io/openshift-migration-best-practices/01-planning.html[OpenShift Migration Best Practices]. Because you must migrate to {product-title} 4, you can use any type of production cluster installation process to create your new cluster.
+
+[id="installing-preparing-existing-components"]
+=== How many existing components do you want to use with your cluster?
+
```

Comment:
I think this section would benefit from some dot titles to help readers scan the content more easily. For example you could include two dot titles as follows:

```
.Customizing installer-provisioned infrastructure installations
.Installing {product-title} by using user-provisioned infrastructure
```
The first of those could go here. I have added another comment where the second title could potentially go.